base/uk.ac.stfc.isis.ibex.ui.blocks/src/uk/ac/stfc/isis/ibex/ui/blocks/groups/GroupsPanel.java

```diff
@@ -55,10 +55,11 @@
 	
     protected static final Font MESSAGE_FONT = SWTResourceManager.getFont("Arial", 14, SWT.NORMAL);
     private static final String DISCONNECTED_MESSAGE = "IBEX SERVER DISCONNECTED\nPlease talk to your point of contact";
-    private static final String CONNECTED_NO_GROUPS_MESSAGE = "IBEX SERVER CONNECTED:\nNo Groups Present";
+    private static final String CONNECTED_NO_GROUPS_MESSAGE =
+            "IBEX SERVER CONNECTED:\nNo Groups Present\nNew Blocks can be added by going to Configuration->Edit Current Configuration->Blocks->Add Blocks";
```

Comment:
The last line displayed is quite long, maybe it should be split across two lines by adding a \n between "...going to" and "Configuration->..."

Reply:
Done, I have split that long instruction into two lines.